symphony-bdk-core-invokers/symphony-bdk-core-invoker-jersey2/pom.xml

```diff
@@ -142,6 +142,10 @@
             <artifactId>mockserver-netty</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```

Comment:
we can use the provided scope to avoid exposing it (https://projectlombok.org/setup/maven)